src/schemaToPropTypes.js

```diff
@@ -181,13 +230,18 @@ const propertiesReducer = (properties, requiredProps) => (str, propertyName) =>
  * @returns {string}
  */
 const getPropTypes = (schemaName, schema) => {
+	const isObjectDefinition = schema.type === 'object';
 	const requiredProps = 'required' in schema && schema.required;
-	const propTypeStringIndented = useIndentation(propTypeString);
-	const reducer = propertiesReducer(schema.properties, requiredProps);
+	const propTypeStringIndented = useIndentation(
+		propTypeString,
+		!!isObjectDefinition,
```

Comment:
`isObjectDefinition` is already a boolean, why are you casting it again to one? Or am I missing something?